clients/cli/src/main/java/org/apache/gravitino/cli/commands/UntagEntity.java

```diff
@@ -91,24 +93,20 @@ public void handle() {
     } catch (NoSuchCatalogException err) {
       exitWithError(ErrorMessages.UNKNOWN_CATALOG);
     } catch (NoSuchSchemaException err) {
-      exitWithError(ErrorMessages.UNKNOWN_TABLE);
+      exitWithError(ErrorMessages.UNKNOWN_SCHEMA);
     } catch (NoSuchTableException err) {
       exitWithError(ErrorMessages.UNKNOWN_TABLE);
     } catch (Exception exp) {
       exitWithError(exp.getMessage());
     }
 
-    String all = String.join(",", removeTags);
-
-    if (all.equals("")) {
-      all = "nothing";
-    }
-
-    if (tags.length > 1) {
-      System.out.println(
-          entity + " removed tags " + String.join(",", tags) + " now tagged with " + all);
-    } else {
-      System.out.println(entity + " removed tag " + tags[0].toString() + " now tagged with " + all);
-    }
+    System.out.println(
+        entity
+            + " removed tag(s): ["
+            + COMMA_JOINER.join(tags)
+            + "], now tagged with "
+            + "["
+            + COMMA_JOINER.join(removeTags)
+            + "]");
```

Comment:
The output has changed when no tags have been removed, please change sback so it says "nothing".

Reply:
This also needs to be consistent with the messages given when you add tags.